src/PlatesRendererFactory.php

```diff
@@ -39,11 +43,17 @@ class PlatesRendererFactory
     public function __invoke(ContainerInterface $container)
     {
         $config = $container->has('config') ? $container->get('config') : [];
-        $config = isset($config['templates']) ? $config['templates'] : [];
 
         // Create the engine instance:
         $engine = new PlatesEngine();
 
+        // Enable assets extension
+        if (isset($config['plates']['assets_path'])) {
+            $engine->loadExtension(new PlatesExtension($config['plates']['assets_path']));
```

Comment:
I'm wondering if it might make sense to also allow loading of arbitrary extensions, like we [did in Twig](https://github.com/zendframework/zend-expressive-twigrenderer/blob/master/src/TwigRendererFactory.php#L107-L136)? We could still load the PlatesExtension as done here — or provide a factory for it — but that might provide a bit more flexibility with regards to this factory.

Thoughts?